### PR TITLE
webui: commonLanguages can contains codes that are not in the locales the API returns

### PR DESCRIPTION
**Pull request by KKoukiou**
Fixes crash in the language screen as described here https://issues.redhat.com/browse/INSTALLER-3504

### CODE REVIEW

File: ui/webui/src/components/localization/InstallationLanguage.jsx [hunk 1]
```diff
@@ -178,9 +179,10 @@ class LanguageSelector extends React.Component {
                       key="group-common-languages"
                     >
                         {
-                            this.state.commonLocales.map(locale => {
-                                return createMenuItem(findLocaleWithId(locale), "option-common-");
-                            })
+                            this.state.commonLocales
+                                    .map(findLocaleWithId)
+                                    .filter(locale => locale)
+                                    .map(locale => createMenuItem(locale, "option-common-"))
```
Comment (jelly) [inline ui/webui/src/components/localization/InstallationLanguage.jsx:185]: honestly feels that we iterate this a bit too much, but this page needs a bit of a re-design. Like it feels more sensible to do this once during initialisation.

**Review by jelly — APPROVED** [review]

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by rvykydal — APPROVED** [review]
Thank you.

**Review by M4rtinK — APPROVED** [review]
Thanks a lot! :)

**Pull request merged**